packages/playground/src/views/jitsi_view.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <view-layout>
+    <tf-jitsi />
+
+    <template #list>
+      <TfDeploymentList title="Jitsi Instances" :project-name="name" />
```

Comment:
```suggestion
      <TfDeploymentList title="Jitsi" :project-name="name" />
```

Just to match other solutions.